index.js

```diff
@@ -4,7 +4,16 @@ module.exports = {
   },
   "extends": "eslint:recommended",
   "rules": {
-    "strict": "warn",
+    "prefer-const": "warn",
+    "no-var": "warn",
+    "prefer-template": "warn",
+    "prefer-arrow-callback": "error",
```

Comment:
Can we add in this option: `allowNamedFunctions: true` here? If we want a stack trace for some callback it'd be nice to be allowed to do that, but anonymous func callbacks are probably fine to be arrow funcs always